### ISSUE TITLE
describe_vpc_classic_link Implementation

### ISSUE THREAD

**Issue by ricfeatherstone** [spulec/moto#2902]
Are there plans to implement `describe_vpc_classic_link`?

I'm trying to test `terraform` code to create a `vpc` and running into the issue described in #2551 

**Terraform Config**

```
resource aws_vpc target {
  cidr_block = "10.0.0.0/16"
}

provider aws {
  version = "~> 2.53"
  region  = "eu-west-1"

  endpoints {
    ec2 = "http://127.0.0.1:5000/"
  }
}
```
**Steps**

1. Start server `moto_server ec2`
1. Run Terraform `terraform apply -auto-approve`

**Moto Server Output**

```
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
127.0.0.1 - - [17/Apr/2020 16:09:10] "POST  HTTP/1.1" 200 -
127.0.0.1 - - [17/Apr/2020 16:09:15] "POST  HTTP/1.1" 200 -
127.0.0.1 - - [17/Apr/2020 16:09:15] "POST  HTTP/1.1" 200 -
127.0.0.1 - - [17/Apr/2020 16:09:15] "POST  HTTP/1.1" 200 -
127.0.0.1 - - [17/Apr/2020 16:09:15] "POST  HTTP/1.1" 200 -
127.0.0.1 - - [17/Apr/2020 16:09:15] "POST  HTTP/1.1" 200 -
127.0.0.1 - - [17/Apr/2020 16:09:15] "POST  HTTP/1.1" 200 -
127.0.0.1 - - [17/Apr/2020 16:09:15] "POST  HTTP/1.1" 200 -
127.0.0.1 - - [17/Apr/2020 16:09:15] "POST  HTTP/1.1" 500 -
Error on request:
Traceback (most recent call last):
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/werkzeug/serving.py", line 323, in run_wsgi
    execute(self.server.app)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/werkzeug/serving.py", line 312, in execute
    application_iter = app(environ, start_response)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/moto/server.py", line 135, in __call__
    return backend_app(environ, start_response)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/moto/core/utils.py", line 144, in __call__
    result = self.callback(request, request.url, {})
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/moto/core/responses.py", line 197, in dispatch
    return cls()._dispatch(*args, **kwargs)
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/moto/core/responses.py", line 295, in _dispatch
    return self.call_action()
  File "/home/ric/git/japeoh/zone-tfmodule/.venv/lib/python3.7/site-packages/moto/core/responses.py", line 393, in call_action
    "The {0} action has not been implemented".format(action)
NotImplementedError: The describe_vpc_classic_link action has not been implemented
```

**Comment by anardy** [comment]
I think this issue could be closed
The method has already been implemented [describe_vpc_classic_link](https://github.com/spulec/moto/blob/80b64f9b3ff57515db1fc07329bf8e5f519597aa/moto/ec2/responses/vpcs.py#L88) and [tested](https://github.com/spulec/moto/blob/80b64f9b3ff57515db1fc07329bf8e5f519597aa/tests/test_ec2/test_vpcs.py#L717)

**Comment by bblommers** [comment]
Which version of moto are you running, @ricfeatherstone? This should be available in moto >= 1.3.15.dev70

**Comment by ricfeatherstone** [comment]
Thanks, was using 1.3.14. 
With 1.3.15.dev70 DescribeVpcClassicLink is now returning a response.
Terraform is now repeatedly calling DescribeVpcClassicLinkDnsSupport which is also returning a response, so need to investigate.
